source/developer/fx-guidelines.rst

```diff
@@ -248,6 +248,16 @@ The color of the on_hover state should be the same as in active state, with opac
 |
 ---------------------------
 
+Display: Tooltips
+---------------------------
+
+Tooltips text can be title case or sentence case.
```

Comment:
```suggestion
Tooltip text may be title case or sentence case depending on their context.
```

Reply:
Shouldn't this be, depending on its context, as a "Tooltip" is singular.
Or we can use "Tooltips" and then use "their".

Reply:
@jasonblais ^

Reply:
@asaadmahmood Yes, good catch. 

```
Tooltip text may be title case or sentence case depending on its context.
```

Reply:
Makes sense.